cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -1319,3 +1320,166 @@ def test_update_committee_threshold_out_of_range(
+
+                rat_action = governance_utils.lookup_ratified_actions(
+                    gov_state=rat_gov_state, action_txid=action_txid
+                )
+                assert rat_action, "Action not found in ratified actions"
+
+                # Wait for enactment
+                _cur_epoch = cluster.wait_for_new_epoch(padding_seconds=5)
+
+            # Check enactment
+            assert _cur_epoch == fin_approve_epoch + 2, f"Unexpected epoch {_cur_epoch}"
+            enact_gov_state = cluster.g_conway_governance.query.gov_state()
+            conway_common.save_gov_state(
+                gov_state=enact_gov_state, name_template=f"{temp_template}_enact_{_cur_epoch}"
+            )
+
+            prev_action_rec = governance_utils.get_prev_action(
+                action_type=action_type,
+                gov_state=enact_gov_state,
+            )
+
+            assert prev_action_rec.txid == action_txid, "Action not enacted"
+
+        deposit_amt = cluster.conway_genesis["govActionDeposit"]
+        anchor_url = "http://www.cc-update.com"
+        anchor_data_hash = "5d372dca1a4cc90d7d16d966c48270e33e3aa0abcb0e78f0d5ca7ff330d2245d"
+        prev_action_rec = governance_utils.get_prev_action(
+            action_type=governance_utils.PrevGovActionIds.COMMITTEE,
+            gov_state=cluster.g_conway_governance.query.gov_state(),
+        )
+
+        # Set CC threshold to zero
+        update_threshold_action = cluster.g_conway_governance.action.update_committee(
+            action_name=temp_template,
+            deposit_amt=deposit_amt,
+            anchor_url=anchor_url,
+            anchor_data_hash=anchor_data_hash,
+            threshold="0",
+            prev_action_txid=prev_action_rec.txid,
+            prev_action_ix=prev_action_rec.ix,
+            deposit_return_stake_vkey_file=pool_user_lg.stake.vkey_file,
+        )
+
+        tx_files = clusterlib.TxFiles(
+            proposal_files=[update_threshold_action.action_file],
+            signing_key_files=[
+                pool_user_lg.payment.skey_file,
+            ],
+        )
+
+        tx_output_action = clusterlib_utils.build_and_submit_tx(
+            cluster_obj=cluster,
+            name_template=f"{temp_template}_update_threshold",
+            src_address=pool_user_lg.payment.address,
+            use_build_cmd=True,
+            tx_files=tx_files,
+        )
+
+        action_txid = cluster.g_transaction.get_txid(tx_body_file=tx_output_action.out_file)
+        action_gov_state = cluster.g_conway_governance.query.gov_state()
+        _cur_epoch = cluster.g_query.get_epoch()
+        conway_common.save_gov_state(
+            gov_state=action_gov_state, name_template=f"{temp_template}_action_{_cur_epoch}"
+        )
+        prop_action = governance_utils.lookup_proposal(
+            gov_state=action_gov_state, action_txid=action_txid
+        )
+        action_ix = prop_action["actionId"]["govActionIx"]
+
+        conway_common.cast_vote(
```

Comment:
Make sure you manage to cast the votes in one epoch (ie. you are not close to epoch boundary).